python-libraries/nanover-openmm/src/nanover/openmm/cli.py

```diff
@@ -63,6 +63,20 @@ def handle_user_arguments() -> argparse.Namespace:
         metavar="STEPS",
         help="Update the interactions every STEPS dynamics steps.",
     )
+    parser.add_argument(
+        "-q",
+        "--include-velocities",
+        type=bool,
```

Comment:
Replace `type=bool` with
```suggestion
        action="store-true",
```